dali/bus.py

```diff
@@ -140,4 +140,4 @@ def assign_short_addresses(self):
                     i.send(gear.Terminate())
                     raise NoFreeAddress()
                 low = low + 1
-        i.send(gear.Terminate())
+        i.send(gear.Terminate())
```

Comment:
No need to change this file.

Reply:
.dll is required in windows. The original HID library works only in Linux. Using hidapi.dll the code works both in Linux and in Windows. Maybe a drivers folder would be a better place.

Reply:
The .dll is compiled from here https://github.com/signal11/hidapi

Reply:
And in Linux its a dependency

Reply:
The code you linked to at https://github.com/signal11/hidapi is a C library with no Python wrapper.  I can't find a Python library corresponding to the hidapi calls you've written; both the common ones (https://github.com/trezor/cython-hidapi and https://github.com/jbaiter/hidapi-cffi) have different APIs.

Reply:
see the readme
https://github.com/hasseb/python-dali/tree/master/gui

Reply:
Oh, I see!  There's yet another one!